geoportal/geoportailv3_geoportal/lib/lux_authentication.py

```diff
@@ -22,16 +19,30 @@ def create_authentication(settings):
     http_only = http_only.lower() in ("true", "yes", "1")
     secure = settings.get("authtkt_secure", "True")
     secure = secure.lower() in ("true", "yes", "1")
-    cookie_authentication_policy = AppAwareAuthTktAuthenticationPolicy(
+
+    # AuthenticationPolicy for login via the app
+    app_authentication_policy = ConditionalAuthTktAuthenticationPolicy(
```

Comment:
I'll rename "app" to "mobile app" (maybe just the comment) for clarity.

Reply:
I've updated the comment